src/app/setup-wizard/pre-visit/pre-assessment-setup/pre-assessment-setup.service.ts

```diff
@@ -45,7 +73,7 @@ export class PreAssessmentSetupService {
           let camelCaseType = trimmedType.charAt(0).toLowerCase() + trimmedType.slice(1);
           let convertedCost = this.convertValue.convertValue(
```

Comment:
Converting costs aren't working properly. as a rule of thumb, if you change the energy unit the resulting cost should not change if the conversions are done correctly